website/docs/d/is_lb_pool_member.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_lb_pool_member"
+description: |-
+  Get information about LoadBalancerPoolMember
+---
+
+# ibm_is_lb_pool_member
+
+Provides a read-only data source for LoadBalancerPoolMember.
+
+## Example Usage
+
+```terraform
+data "ibm_is_lb_pool_member" "example" {
+	member = element(split("/",ibm_is_lb_pool_member.example.id),2)
+	lb = ibm_is_lb.example.id
+	pool = element(split("/",ibm_is_lb_pool.example.id),1)
```

Comment:
ibm_is_lb_pool as attribute pool_id which gives the pool id directly ....instead of split function

check all docs examples

Reply:
Yes, Thank you for noticing, made the required changes